test/jdk/java/lang/Float/Binary16ConversionNaN.java

```diff
@@ -71,13 +77,27 @@ private static boolean isNaN(short binary16) {
             && ((binary16 & 0x03ff) != 0 );    // significand nonzero.
     }
 
-    private static int testRoundTrip(int i) {
+    private static int sign(short binary16) {
+        return (binary16 & 0xf000);
```

Comment:
I think this should read
`return binary16 & 0x8000;`

Reply:
Further, if the only modified bit is the quier/signaling bit the test could be more specific

Reply:
Test is reverted, so there is nothing to be done here :)